code/game/mecha/mech_bay.dm

```diff
@@ -21,9 +22,12 @@
 	circuit = /obj/item/circuitboard/machine/mech_recharger
 	var/obj/mecha/recharging_mech
 	var/obj/machinery/computer/mech_bay_power_console/recharge_console
-	var/max_charge = 50
+	///Power unit per second to charge by, modified by /RefreshParts()
+	var/recharge_power = 25
```

Comment:
Why did this change

Reply:
I added the reason in the comment. RefreshParts() overrides this default 50 with a 25.